### PR TITLE
Milestone 2: Works section

### PR DESCRIPTION
**Pull request by theodoku**
1. Create a new branch
2. Build the works section for portfolio
3. Create the about section
4. Style sections using the Figma template 1

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -105,3 +114,301 @@ p {
...
+  border-radius: 16px;
+  min-height: 90%;
+  padding: 16px;
+  margin: 0 24px;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+}
+
+.project-name {
```
Comment (bienvenuushindi) [inline style.css:137]: :red_circle: Kindly make sure this CSS selector is applied on the right element(p-title) to match the Figma Design
It seems your p-title elements inherit the default font-size assigned by the browser.

Your Design (1.5rem or 48px is the current font-size applied to this element )
your title

Figma Design (32px)
figma

An alternative solution would be to apply this font-size directly to p-title selector as follow :arrow_down: 

```
.p-title {
    font-size: 32px;
}
```

File: index.html [hunk 2]
```diff
@@ -78,5 +78,243 @@ <h2><a href="#">LET’S CONNECT</a></h2>
...
+                </div>
+                <h4>HTML</h4>
+              </li>
+              <li class="lang-skills">
+                <div class="langImg">
+                  <img src="images/css.png" id="langImag" alt="css" />
+                </div>
+                <h4>CSS</h4>
+              </li>
+            </ul>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: For semantics reason, Kindly make sure you wrap this block in a li tag as only li tag should be direct child of ul tag.

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
Hi @theodoku  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks good:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @theodoku, 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**